deployment/kubernetes/aerie-gateway-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: aerie-dev
```

Comment:
Just curious about the process here. Is there an easy way to change the `namespace` if we want to deploy to another venue at some point?

Reply:
As mentioned in the "Future Work" section, I'd like to implement a configuration mechanism like [kustomize](https://kustomize.io/), which is built in to `kubectl`. It would allow us to maintain patch files for dev/prod and apply them to respective environments.

e.g.
```sh
cat kustomization.yaml
```
```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

# list of manifests to be applied
resources:
- deployment/kubernetes

# namespace to deploy all Resources to
namespace: aerie-prod
```

This would patch every manifest in our deployment dir, and could change namespaces, replica counts, environment variables, secrets, etc. Aerie users could then just maintain this one patch/overlay file, and apply it on top of our base manifests.